s32230-MiloszKurszewski/zadania3/zadania3.txt

```diff
@@ -0,0 +1,210 @@
+Ćwiczenia 1
+Napisz skrypt, który sprawdzi, czy podano więcej niż jeden parametr. J
+eśli tak, to niech wyświetli pierwszy parametr. Jeśli nie, 
+to niech wyświetli komunikat, że nie podano parametrów.
+
+Skrypt:
+#!/bin/bash
+Argumenty= $#
+if ["$Argumenty" -gt 1]; then
+   echo "$1"
+elif [ "$Argumenty" -eq 0 ]; then
+   echo "Nie podano żadnych parametrów"
+else 
+   echo "Podano tylko jeden argument"
+fi
+Wniosek: Skrypt działa, instrukcje warunkowe w bashu są łatwe w konstrukcji, żeby sprawdzić
+czy podano więcej niż jeden parametr użyłem dwóch poleceń
+-gt sprawdza czy liczba parametrów w $# jest większa niż 1
+-eq sprawdza czy liczba parametrów w $# jest równa 0, żeby wypisać że nie podano żadnych
+else na podstawie warunku że, parametrów nie jest więcej niż 2 i nie są równe zeru, wypisuje
+ze zostal podany jeden.
+
+
+#bin/bash
+
+Napisz skrypt, który:
+
+sprawdzi, czy istnieje plik config.cfg w bieżącym katalogu.
+
+Jeśli taki plik istnieje, to niech go wykona.
+
+Jeśli nie istnieje, to niech sprawdzi, czy podano argument i wykona plik o nazwie takiej jak
+wartość argumentu (parametru) skryptu.
+
+Jeśli żadne z powyższych nie nastąpiło, to niech wyświetli komunikat o błędzie.
+
+Skrypt:
+#!/bin/bash
+
+if [ -f "config.cfg" ]; then
+    ./config.cfg
+elif [ "$#" -gt 0 ]; then
+    ./"$1"
+else
+    echo "Błąd"
+fi
+Wniosek: Wykonałem skrypt o bardzo prostej strukturze,
+zastosowałem logikę z poprzedniego zadania.
+Jeżeli w katalogu znajduje się config.cfg to zostanie wykonany,
+jeżeli nie znajduje się, a ilość parametrów w skrypcie jest większa od 0,
+to pierwszy parametr zostanie wykonany.
+Jeżeli żadne z tych warunków się nie sprawdzi, zostanie wypisany błąd.
+Napisz skrypt, który:
```

Comment:
zlewa się to z poprzednim rozwiązaniem/zadaniem.

Ogólnie to dobrze by było gdyby to porozbijać na poszczególne pliki z zadaniami.